events/fair-ml.html

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: "Fair ML Speaker Series"
+permalink: "/fair-ml"
+---
+
+<h1 class="title page-title"> Fair ML Speaker Series</h1>
+
+<p style = "font-size: 115%;">
+	Thanks to our <a class="tla-link" href="{{site.baseurl}}/speaker-series"> Speaker Series Team </a>, our Fair ML Speaker Series was a success! <b> Learn all about it on this page. </b>
+
+</p>
```

Comment:
I think this content should instead generally explain what the Fair ML speaker series is. You don't necessarily have to write the copy - feel free to reach out to someone in the slack channel for help on this!